website/content/labs/lab1.md

```diff
@@ -2,26 +2,39 @@
-draft: true
+draft: false
 ---
 
-**Due:** Thursday, April 11th, 2:30pm
+**Due:** Wednesday, April 15th, 8pm
 
 {{% warning %}}
 **NOTE**: This lab is based on a Linux and Git lab originally written by Anne Rogers for CMSC 12100 (and updated and edited by a variety of instructors and TAs throughout the years)
 {{% /warning %}}
 
-In some of your previous CS courses, you became familiar with using Subversion (SVN) to handle revision control (or *version control*) of your projects and assignments, and to submit your work for grading. In this class, we will be using Git, a more widespread version control system that is more commonly used in a variety of software projects.
+In some of your previous CS courses, you may have become familiar with using Subversion (SVN) to handle revision control (or *version control*) of your projects and assignments, and to submit your work for grading. In this class, we will be using Git, a more widespread version control system that is more commonly used in a variety of software projects.
 
 Like SVN, Git is a *version control system* that maintains files in a *repository* that contains not just files, but also a record of all the changes made to those files. Git tracks every version of a file or directory using *commits*. When you have made changes to one or more files, you can logically group those changes into a "commit" that gets added to your repository. You can think of commits as “checkpoints” in your work, representing the work you’ve done since the previous checkpoint. This mechanism makes it possible to look at and even revert to older versions of a file by going back to your code as it was when you “checkpointed” it with a commit.
 
-You will each have a personal Git repository that is hosted on a central server run by the CS department (the software running on that server is [GitLab](https://about.gitlab.com/), so we will often refer to it as "the GitLab server"). You may have also heard of [GitHub](https://github.com/), a web-based hosting service for Git repositories. We will be using GitHub later in the quarter for the course project, but will be using GitLab for the labs, since it will be easier for us to access your work there (since it is tied to CNetID authentication, etc.). We will also be browsing some Git repositories hosted on GitHub, but you will not need a GitHub account just yet.
+For each lab assignment, a Git repository will be created for you on [GitHub](https://github.com/), a web-based hosting service for Git repositories. However, before that repository can be created for you, you need to have a GitHub account. If you do not yet have one, you can get an account here: https://github.com/join. Once you create your account, you may want to get the [Student Developer Pack](https://education.github.com/pack), which will give you access to a lot of other features (please note that having the Student Developer Pack is not necessary for CS 220; it's just a nice benefit you get as a student) 
```

Comment:
```suggestion
For each lab assignment, a Git repository will be created for you on [GitHub](https://github.com/), a web-based hosting service for Git repositories. However, before that repository can be created for you, you need to have a GitHub account. If you do not yet have one, you can get an account here: https://github.com/join. Once you create your account, you may want to get the [Student Developer Pack](https://education.github.com/pack), which will give you access to a lot of other features (please note that having the Student Developer Pack is not necessary for CS220; it's just a nice benefit you get as a student) 
```

Reply:
idk why it's not highlighting the suggestion - all I did was remove the space between `CS` and `220` (i.e. `CS 220` -> `CS220`)

Reply:
That is inconsistent with how we write the course name every where else, including on the front page of the website:

![image](https://user-images.githubusercontent.com/448918/78728253-7d72f900-78fc-11ea-86d5-cce63236fa48.png)